package.json

```diff
@@ -34,6 +34,7 @@
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
-    "kind-of": "^6.0.3"
+    "kind-of": "^6.0.3",
+    "node-forge": "^0.10.0"
   }
 }
```

Comment:
- name: Setting context for Kubernetes cluster of Alibaba Cloud Kubernetes Service (ACK)
  uses: aliyun/ack-set-context@v1

Reply:
How